app/assets/stylesheets/views/comments.scss

```diff
@@ -275,3 +275,59 @@
 .moderator-template-button:not(.active) {
   @extend .crayons-btn--outlined;
 }
+
+#toggle-comments-sort-dropdown {
+  cursor: pointer;
+}
+
+#comments-sort-dropdown-container {
+  z-index: var(--z-dropdown);
+
+  .comments-sort-dropdown {
+    left: 0;
+    padding: var(--su-4);
+    padding-right: var(--su-6);
+    width: 310px;
+
+    &__list {
+      list-style-type: none;
+    }
+  }
```

Comment:
refactor (blocking) - I think almost all of these you don't need, as they're applied by the `crayons-dropdown` class. The only one I think you require is the padding, and you can add this with a helper classname directly on the HTML element - `p-4`

Reply:
good catch. But without the width property, the end result is a bit different from what was posted in the issue.
<img width="468" alt="Screenshot 2022-03-26 at 2 10 38 PM" src="https://user-images.githubusercontent.com/9512872/160231846-4b045894-8280-4ef4-9459-e77d6d4cb649.png">